frontend/src/routes/(authenticated)/project/create/+page.ts

```diff
@@ -35,16 +35,30 @@ export async function load(event: PageLoadEvent) {
 
     requestingUser = userResultsPromise.data?.users?.items?.[0];
   }
-  const myOrgsPromise = await client.query(graphql(`
-        query loadMyOrgs {
-            myOrgs {
-                id
-                name
-            }
-        }
-    `), {}, { fetch: event.fetch });
-  const myOrgs = myOrgsPromise.data?.myOrgs;
-  return { requestingUser, myOrgs };
+
+  if (userIsAdmin) {
```

Comment:
It looks like this block of code can be refactored, but I am not sure how it can be done.

Reply:
I agree. I'd make a variable `orgs` outside of the if block, then inside each block assign a value to orgs from the resulting query. Then after the if block you can  write `return { requestingUser, myOrgs: orgs };` once